cocos2d/core/base-nodes/CCNodeCanvasRenderCmd.js

```diff
@@ -368,24 +371,50 @@ cc.Node.RenderCmd.prototype = {
     },
 
     visitChildren: function(){
```

Comment:
It's for setting correctly the vertexZ as depth value which will be used in the matrix.

Previously, the vertexZ of a Node is always 0